testplugins/src/main/java/org/spongepowered/test/entity/DisplayEntityTest.java

```diff
@@ -224,8 +226,19 @@ public void onRegisterCommand(final RegisterCommandEvent<Parameterized> event) {
                         blockDisplay.offer(Keys.INTERPOLATION_DURATION, Ticks.of(20));
                         blockDisplay.offer(Keys.INTERPOLATION_DELAY, Ticks.of(20));
 
-                        textDisplay = createEntity(player.world(), EntityTypes.TEXT_DISPLAY, centerPos, forwardDir, col6, 0);
-                        textDisplay.offer(Keys.DISPLAY_NAME, Component.text("Look at this interpolation").color(NamedTextColor.RED));
+                        blockDisplay = createEntity(player.world(), EntityTypes.BLOCK_DISPLAY, centerPos, forwardDir, col6, 0);
+                        blockDisplay.offer(Keys.BLOCK_STATE, BlockTypes.OBSIDIAN.get().defaultState());
+                        blockDisplay.offer(Keys.TRANSFORM, Transform.of(blockCenterOffset)); // set initial value before spawning
+                        blockDisplay.offer(Keys.SHADOW_RADIUS, 2d); // set initial value before spawning
+                        blockDisplay.offer(Keys.SHADOW_STRENGTH, 5d); // set initial value before spawning
+                        player.world().spawnEntity(blockDisplay);
+                        blockDisplay.offer(Keys.TELEPORT_DURATION,Ticks.of(20));
+                        blockDisplay.offer(Keys.INTERPOLATION_DELAY,Ticks.of(20));
```

Comment:
Might as well fix this while at it.
```suggestion
                        blockDisplay.offer(Keys.TELEPORT_DURATION, Ticks.of(20));
                        blockDisplay.offer(Keys.INTERPOLATION_DELAY, Ticks.of(20));
```